src/elementary_operators.jl

```diff
@@ -191,6 +192,89 @@ function identity_operator(s::IndsNetworkMap; kwargs...)
   return ITensorNetwork(Op("I"), operator_inds)
 end
 
+" Create an operator bitstring corresponding to the number x"
+function point_to_opsum(s::IndsNetworkMap, x::Number, dim::Int)
+  ttn_op = OpSum()
+  ind_to_ind_value_map = calculate_ind_values(s, x, dim)
+  string_sites = [
+    (ind_to_ind_value_map[only(s[v])] == 1) ? ("Dup", v) : ("Ddn", v) for
+    v in dimension_vertices(s, dim)
+  ]
+  add!(ttn_op, 1.0, (string_sites...)...)
+  return ttn_op
+end
+
+" Create an operator which maps a function to 0 at all points in xs"
+function map_to_zero_operator(
```

Comment:
I think the following would implement this function in a more transparent way

```
function map_to_zero_operator(
    s::IndsNetworkMap, xs::Vector, dims::Vector=[1 for _ in xs]; truncate_kwargs...
  )
    udims = unique(dims)
    @assert length(udims) <= 2 # TODO: generalize 

    dim1_xs = [xs[j] for j in [i -> dims[i] == first(udims), dims]]
    psi_x = reduce(+, [delta_xyz(s, [x], [first(udims)]) for x in dim1_xs])
    psi_x = const_itn(s; c= -1.0) * psi_x

    length(udims) == 1 && return const_itn(s) + psi_x

    dim2_xs = [xs[j] for j in [i -> dims[i] == last(udims), dims]]
    psi_y = reduce(+, [delta_xyz(s, [x], [last(udims)]) for x in dim2_xs])
    psi_y = const_itn(s; c= -1.0) * psi_y
    psi = const_itn(s) + psi_x + psi_y + psi_x * psi_y
    return proj_operator(psi)
end
```

This avoids the use of `OpSum` (which we know can be finicky) and it is clear how to generalise